src/api/entities/types.ts

```diff
@@ -35,3 +35,4 @@ export * from './SecurityToken/types';
 export * from './Venue/types';
 export * from './Instruction/types';
 export * from './Portfolio/types';
+export * from './Proposal/types';
```

Comment:
Everything related to Governance is commented. Should we comment it too?

Reply:
Sure, I'll comment it and any imports that depend on it